ClientApp/app/_layout.tsx

```diff
@@ -6,6 +6,7 @@ export default function RootLayout() {
     <Stack initialRouteName="login">
       <Stack.Screen name="auth/login" options={{ headerShown: false }} />
       <Stack.Screen name="auth/register" options={{ headerShown: false }} />
+      <Stack.Screen name="tabs" options={{ headerShown: false }} />
```

Comment:
Since we are using expo router, it uses the directories as routes. We want the users to navigate to a route that resembles "/home". To achieve this we would need to rename the "tabs" folder to "home". In that directory we can organize our files with a sub folder called "(tabs)". Expo router ignores all folders with parentheses and excludes them from the routes.

Reply:
Noted, to be adjust in the next iteration! Thanks